samples/interactive-tutorials/product/add_fulfillment_places.py

```diff
@@ -52,27 +53,29 @@ def get_add_fulfillment_request(
     return add_fulfillment_request
 
 
-# add fulfillment places to product
 def add_fulfillment_places(product_name: str, timestamp, place_id):
     add_fulfillment_request = get_add_fulfillment_request(
         product_name, timestamp, place_id
     )
-    ProductServiceClient().add_fulfillment_places(add_fulfillment_request)
+    return ProductServiceAsyncClient().add_fulfillment_places(add_fulfillment_request)
 
-    # This is a long running operation and its result is not immediately present with get operations,
-    # thus we simulate wait with sleep method.
-    print("---add fulfillment places, wait 90 seconds :---")
-    time.sleep(90)
 
+async def add_places(product_name: str):
```

Comment:
Should we merge both add_fulfillment_places and add_places to make it look more clear?

Reply:
Yes. merged